pkg/cmd/build/build.go

```diff
@@ -287,10 +287,19 @@ func OptsFromBuildInfo(manifestName, svcName string, b *model.BuildInfo, o *type
 	optionsBuildArgs := map[string]string{}
 	for _, arg := range o.BuildArgs {
 		splittedArg := strings.SplitN(strings.Trim(arg, "\""), "=", 2)
-		optionsBuildArgs[splittedArg[0]] = splittedArg[1]
-		args = append(args, model.BuildArg{
-			Name: splittedArg[0], Value: splittedArg[1],
-		})
+		if len(splittedArg) == 1 {
```

Comment:
I understand that other cases are covered in this if condition like `"MY-KEY="` but `--build-arg MY-KEY` should be managed as `"MY-KEY="`? It is not more appropriate to log that is invalid or is it something common to set an empty key?

Reply:
I wouldn't fail. It's valid on docker and since we are trying to emulate the same experience but running it on remote I'd  support it. WDYT?

Reply:
then go ahead! 👍